packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -741,7 +741,8 @@ export function mapFrameToDataGrid({
             timeRange={timeRange ?? getDefaultTimeRange()}
             height={defaultRowHeight}
             justifyContent={justifyColumnContent}
-            rowIdx={rowIdx}
+            rowIdx={sortedRows[rowIdx].__index}
+            sortedRows={sortedRows}
```

Comment:
probably don't need `sortedRows`, right? since __index is the correct offset into the `field.values`

Reply:
Yep, removed!